tests/test_dock_layout.py

```diff
@@ -69,3 +70,131 @@ def test_validation_dock_layout2(enaml_qtbot, enaml_sleep):
         win.area.layout = HSplitLayout('item1', 'item2', 'item3')
     assert globals() == glob
     enaml_qtbot.wait(enaml_sleep)
+    
+    
+def test_dock_area_interactions(enaml_qtbot, enaml_sleep):
+    """Test interations with the dock area.
+
+    """
+    # Since timers are used the sleep must be greater than the default
+    enaml_sleep = 300
```

Comment:
I am not sure I get why you change the sleep. Furthermore I would prefer to use a conditional (either `or` or an inline `if ... else`) to avoid overwriting the user decision (if I want to look at the interaction I may set the time to 1000.

Reply:
Would a `enaml_sleep = max(300, enaml_sleep)` be sufficient?

Reply:
Yes indeed.